pkg/rbd/rbd_util.go

```diff
@@ -74,7 +75,6 @@ type rbdVolume struct {
 	Monitors           string `json:"monitors"`
 	Pool               string `json:"pool"`
 	DataPool           string
-	ImageFormat        string `json:"imageFormat"`
```

Comment:
I think this is needed for volume created by CSI v1.0.0 driver.

Reply:
Niels, can you please test that we should be able to delete PVC created by csi v1.0.0 driver

Reply:
Deleting an PVC created with an older version works fine.